tools/preseal/Makefile

```diff
@@ -0,0 +1,12 @@
+CC = gcc
+
+all: preseal
+
+preseal:
+	$(Q)$(CC) -o $@ $@.c -lwolftpm
+
+debug:
+	$(Q)$(CC) -o preseal preseal.c -g -lwolftpm -lwolfssl
+
+clean:
```

Comment:
Looks like you added the binary. It should not be tracked and it should be in the .gitignore.